src/core/src/components/formly.field.config.ts

```diff
@@ -32,7 +32,7 @@ export interface FormlyFieldConfig {
    */
   validation?: {
     messages?: {
-      [messageProperties: string]: string | ((error, field: FormlyFieldConfig) => string);
+      [messageProperties: string]: string | ((error: any, field: FormlyFieldConfig) => string);
```

Comment:
if we are going on that way, it would be better to use `noImplicitAny` to make sure all arguments are annotated